docs/tables/{custom_resource_name.group_name}.md

```diff
@@ -0,0 +1,556 @@
+                  type: integer
+      served: true
+      storage: true
+```
+
+```bash
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: temporal-w-spcloudt6t6sk7toegg-tls
+  labels:
+    workspace-id: w_spcloudct6t6sk7toegg
+    identity-id: o_spcloud6un5ogrh1bc0
+    region: apse1
+    shard: "0001"
+    workspace-pluginset: "20220317010004"
+    app.kubernetes.io/component: steampipe-workspace-db
+    app.kubernetes.io/managed-by: steampipe-api
+    app.kubernetes.io/part-of: steampipe-cloud
+    app.kubernetes.io/instance: w-spcloudct6t6sk7toegg
+    app.kubernetes.io/version: 0.13.3-workspace-spcloud.20220317010004
+spec:
+  secretName: temporal-w-spcloudt6t6sk7toegg-tls
+  duration: 87600h # 10 years
+  renewBefore: 360h # 15d
+  usages:
+    - server auth
+    - client auth
+  issuerRef:
+    name: ca-issuer-temporal
+    kind: ClusterIssuer
+  dnsNames:
+    - w-spcloudt6t6sk7toegg
+```
+
+### Inspect the table structure
+
+List all tables:
+
+```sql
+.inspect kubernetes;
++---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| table                                 | description                                                                                                                                                      |
++---------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| certificates.cert-manager.io          | Represents Custom resource certificates.cert-manager.io.                                                                                                         |
+| crontabs.stable.example.com           | Represents Custom resource crontabs.stable.example.com.                                                                                                          |
+| kubernetes_cluster_role               | ClusterRole contains rules that represent a set of permissions.                                                                                                  |
+| kubernetes_cluster_role_binding       | A ClusterRoleBinding grants the permissions defined in a cluster role to a user or set of users. Access granted by ClusterRoleBinding is cluster-wide.           |
+| kubernetes_config_map                 | Config Map can be used to store fine-grained information like individual properties or coarse-grained information like entire config files or JSON blobs.        |
```

Comment:
```suggestion
```suggestion
| ...                                                     | ...                                                                                                                                 |
```

I think we can probably truncate the inspect results a bit here, since all of the table names are not relevant
```